skimage/filters/_window.py

```diff
@@ -0,0 +1,122 @@
+import numpy as np
+from ..transform import warp
+from .._shared.utils import safe_as_int
+from scipy.signal import get_window
+
+
+def window(window_type, size, ndim=2, warp_kwargs=None):
+    """Return an n-dimensional window of a given size and dimensionality.
+
+    Parameters
+    ----------
+    window_type : string, float, or tuple
+        The type of window to be created. Any window type supported by
+        `scipy.signal.get_window` is allowed here. See notes below for a
+        current list, or the SciPy documentation for the version of SciPy
+        on your machine.
+    size : int
+        The size of the window along each axis (all axes will be equal length).
+    ndim : int, optional (default: 2)
+        The number of dimensions of the window.
+    warp_kwargs : dict
+        Keyword arguments passed to `transform.warp` (e.g.,
+        `warp_kwargs={'order':3}` to change interpolation method).
```

Comment:
These need to be double backticks:

```
``warp_kwargs={'order': 3}``
```

One of those annoying differences between ReStructured Text and markdown.

Same for `transform.warp`, or you can use single quotes with a fully-qualified name, ie `skimage.transform.warp`. (Single backticks will be converted to links to the appropriate documentation.)